src/main/kotlin/org/rust/ide/refactoring/RsRefactoringSupportProvider.kt

```diff
@@ -11,15 +11,16 @@ import com.intellij.refactoring.RefactoringActionHandler
 import org.rust.ide.refactoring.extractFunction.RsExtractFunctionHandler
 import org.rust.ide.refactoring.introduceParameter.RsIntroduceParameterHandler
 import org.rust.ide.refactoring.introduceVariable.RsIntroduceVariableHandler
+import org.rust.lang.core.macros.isExpandedFromMacro
 import org.rust.lang.core.psi.RsPatBinding
 import org.rust.lang.core.psi.ext.RsNameIdentifierOwner
 
 class RsRefactoringSupportProvider : RefactoringSupportProvider() {
     override fun isInplaceRenameAvailable(element: PsiElement, context: PsiElement?): Boolean =
-        element is RsPatBinding
+        element is RsPatBinding && !element.isExpandedFromMacro
 
     override fun isMemberInplaceRenameAvailable(element: PsiElement, context: PsiElement?): Boolean =
-        element is RsNameIdentifierOwner
+        element is RsNameIdentifierOwner && !element.isExpandedFromMacro
```

Comment:
Let's add rename tests to check that we allow/disallow renaming in different cases
Also, they would be great integration tests